packages/devtools_app/lib/src/flutter/split.dart

```diff
@@ -214,47 +264,34 @@ class _SplitState extends State<Split> {
             child: DecoratedBox(
               decoration: BoxDecoration(
                 color: Theme.of(context).dividerColor,
-                borderRadius: BorderRadius.circular(Split.dividerMainAxisSize),
+                borderRadius: BorderRadius.circular(Split.defaultSplitterSize),
               ),
               child: SizedBox(
-                height: isHorizontal ? 2.0 : Split.dividerMainAxisSize - 2.0,
-                width: isHorizontal ? Split.dividerMainAxisSize - 2.0 : 2.0,
+                height: isHorizontal ? 2.0 : Split.defaultSplitterSize - 2.0,
+                width: isHorizontal ? Split.defaultSplitterSize - 2.0 : 2.0,
               ),
             ),
           ),
       ],
     );
 
-    final children = <Widget>[];
-    for (int i = 0; i < widget.children.length; i++) {
-      children.addAll([
-        SizedBox(
-          width: isHorizontal ? sizes[i] : width,
-          height: isHorizontal ? height : sizes[i],
-          child: widget.children[i],
-        ),
-        if (i < widget.children.length - 1)
-          GestureDetector(
-            key: widget.dividerKey(i),
-            behavior: HitTestBehavior.translucent,
-            onHorizontalDragUpdate: (details) =>
-                isHorizontal ? updateSpacing(details, i) : null,
-            onVerticalDragUpdate: (details) =>
-                isHorizontal ? null : updateSpacing(details, i),
-            // DartStartBehavior.down is needed to keep the mouse pointer stuck to
-            // the drag bar. There still appears to be a few frame lag before the
-            // drag action triggers which is't ideal but isn't a launch blocker.
-            dragStartBehavior: DragStartBehavior.down,
-            child: SizedBox(
-              width: isHorizontal ? Split.dividerMainAxisSize : width,
-              height: isHorizontal ? height : Split.dividerMainAxisSize,
-              child: Center(
-                child: dragIndicator,
-              ),
-            ),
-          ),
-      ]);
-    }
-    return Flex(direction: widget.axis, children: children);
+    return SizedBox(
+      width: isHorizontal ? Split.defaultSplitterSize : layoutWidth,
+      height: isHorizontal ? layoutHeight : Split.defaultSplitterSize,
+      child: Center(
+        child: defaultDragIndicator,
+      ),
+    );
+  }
+}
+
+void _verifyFractionsSum(List<double> fractions) {
```

Comment:
nit: make the method name indicate what it is you are verifying the fractions sum to.

Reply:
done